sfge/src/main/java/com/salesforce/graph/symbols/apex/ApexStringValue.java

```diff
@@ -273,125 +484,226 @@ public Optional<ApexValue<?>> apply(MethodCallExpressionVertex vertex, SymbolPro
                     INDETERMINANT_VALUE_PROVIDERS.get(methodName).apply(builder).get());
         }
 
+        int numParams = vertex.getParameters().size();
+
         final String value = getValue().orElse(null);
         List<ChainedVertex> parameters = vertex.getParameters();
-        if (METHOD_ABBREVIATE.equalsIgnoreCase(methodName)) {
```

Comment:
A number of methods were handled independently even though they could be handled generically like all the others. So I moved these to be handled generically to match the design of everything else (which will make a refactoring of this code in the future much easier since the pattern is now obvious).